tests/integration/states/pip.py

```diff
@@ -361,7 +364,6 @@ def test_issue_6833_pip_upgrade_pip(self):
             # previously installed
             ret = self.run_function(
                 'pip.install', ['pip==6.0'], upgrade=True,
-                ignore_installed=True,
```

Comment:
@twangboy and I got to the bottom of this earlier today. When this test had failed in the past, the starting pip version has just been updated. But after taking a closer look, I found that the failure was being caused by stale .pyc files left behind. These were left in place by the ``ignore_installed=True`` as it prevents an uninstall from being done on a package if a new version is being installed.

I spoke with @s0undt3ch earlier and he could not recall the reason for using this argument initially. My best guess is that perhaps when the test was first written, pip was not capable of updating itself (i.e. the uninstall which pip performs before it installs the new version could have been breaking the update process). This, however, is only a guess.

TL;DR it's OK to remove the ``ignore_installed=True``.